EC-Setup-SecLog.sh

```diff
@@ -605,6 +605,8 @@ configure_seclog() {
     --profile $seclogprofile \
     --parameters $guarddutyparams
 
+    aws guardduty create-ip-set --detector-id $(aws guardduty list-detectors --output text --query DetectorIds[0]) --name CSIRC_Nessus_EC2 --format TXT --location https://csirc-nessus-scanner-ec2-ip-ranges.s3.eu-west-1.amazonaws.com/ip-ranges.txt --activate
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Quote this to prevent word splitting.